contrib/agate/files/agate

```diff
@@ -0,0 +1,11 @@
+# agate system service
+
+type            = bgprocess
```

Comment:
why is this bgprocess? it doesn't even detach or anything

Reply:
it should be a process